src/main/java/de/hbz/lobid/helper/EtikettMaker.java

```diff
@@ -43,7 +43,7 @@
 
 	private static final String TYPE = "type";
 	private static final String ID = "id";
-	private String contextLocation = "web/conf/context.jsonld";
+	private static String contextLocation = "web/conf/context.jsonld";
```

Comment:
I think this is dangerous, because we now have a static getter and a non-static instance setter for this static field. So we could call the setter on an instance, which we would expect to change only the instance, but would actually change the context for all instances, since it's actually static. Maybe it would be fine to just hard-code the context location for the tests? Or to set up a separate constant, that is used by the instance variable?

Reply:
Agreed. Will (re)use instances instead of introduce statics.